doc/tutorial/code/solutions/Ex05b.fst

```diff
@@ -27,7 +27,7 @@ let rec fib a b n =
   | _ -> fib b (a+b) (n-1)
 
 val fib_is_ok_aux : i:nat -> n:nat{i<=n} ->
-      Tot (u:unit{fib (fibonacci i) (fibonacci (i+1)) (n-i) = fibonacci n})
+      Lemma (fib (fibonacci i) (fibonacci (i+1)) (n-i) = fibonacci n)
```

Comment:
Hello! Thanks for the PR.

F* requires an `ensures` annotation here (as in `(ensures (fib (fibonacci i) (fibonacci (i+1)) (n-i) = fibonacci n))`). If you try to verify this locally, you should see the error message.

If you fix it, I will be happy to merge.

Reply:
Is `ensures` necessary?
I can verify locally:
```$ fstar.exe Ex07b.fst
Verified module: Ex07b (4747 milliseconds)
All verification conditions discharged successfully
```

Reply:
Yes, please try `Ex05b.fst`.

Reply:
Sorry, I didn't see it doesn't work with `decreases`